### DISCUSSION TITLE
How to check payload in browser?

### DISCUSSION THREAD

**Discussion by Quitalizner** [geckosio/geckos.io#246] [Help / Questions]
Websockets allow the payload in/out to be monitored in the network tab.
Geckos.io doesn't seem to do the same.

How do I check the incoming and outgoing payload?

**Comment by yandeu** [comment]
In chrome it's `chrome://webrtc-internals/`

**Reply by Quitalizner** [reply]
I can only see the stats, not the actual data received and sent.
Is there any way to see that?
